src/Host/Monitor/Impl/Logging/MonitorLogger.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.Common.Core.Disposables;
+using Microsoft.Extensions.Logging;
+using System;
+using System.IO;
+
+namespace Microsoft.R.Host.Monitor.Logging {
+    internal sealed class MonitorLogger : ILogger, IDisposable {
```

Comment:
Can we reuse the file logger from the broker? It seems like these two ought to be fundamentally the same.